pkg/bundle/bundle.go

```diff
@@ -215,13 +220,30 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 		Message: message,
 	}
 
+	// Requeue after the minimum poll interval time of all HTTPS bundles
+	requeueAfter := time.Duration(0)
```

Comment:
note: It seems the obvious default now but I had to [check](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.3/pkg/reconcile#Result) that a duration of 0 means "don't requeue" and not "requeue immediately" :grin: